src/cli.ts

```diff
@@ -86,6 +93,23 @@ async function run(argv: string[]) {
   }
 
   const fileMap = await generateDeclarations(args.root, config);
+
+  if (args.deleteExisting) {
```

Comment:
Consider defining a type for `args` and putting it next to the argument descriptors at the top

Reply:
Good idea, done. It would be nice to have a flags library that was typed out of the box, maybe by defining a class with decorators or something.